consul_config.py.ctmpl

```diff
@@ -1,9 +1,9 @@
 {{- define "KEY" -}}
-    {{ key (printf "docker-server-configs/LB/config.%s.json/%s" (env "DEPLOY_ENV") .) }}
+    {{ keyOrDefault (printf "docker-server-configs/LB/config.%s.json/%s" (env "DEPLOY_ENV") .)  "KEYDOESNOTEXIST" }}
```

Comment:
can we make this value include the key itself? so we end up with something like `KEYDOESNOTEXIST_ incoming_exchange`. This allows us to include information about the missing key directly in an error message